server/storage/sqldb.go

```diff
@@ -127,8 +164,9 @@ func (db *SQLStorage) UpdateMany(gun data.GUN, updates []MetaUpdate) error {
 			// called, version ordering in the updates list must be enforced
```

Comment:
Totally non-blocking observation, unrelated to this PR - can we just sort the updates and then call `UpdateCurrent`?  cc @endophage